lib/src/screens/dashboard/dashboard_page.dart

```diff
@@ -132,34 +136,52 @@ class DashboardPage extends BasePage {
                       .display1
                       .backgroundColor),
             )),
-        Container(
-          padding: EdgeInsets.only(left: 45, right: 45, bottom: 24),
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: <Widget>[
-              ActionButton(
-                  image: sendImage,
-                  title: S.of(context).send,
-                  route: Routes.send),
-              ActionButton(
-                  image: exchangeImage,
-                  title: S.of(context).exchange,
-                  route: Routes.exchange),
-              if (!isMoneroOnly)
-                ActionButton(
-                  image: buyImage,
-                  title: S.of(context).buy,
-                  onClick: () async => await _onClickBuyButton(context),
+
+         
+        ClipRect(
+          child:Container(
+          height: 75,
+          width: 343,
```

Comment:
Can we use flex instead of fixed width, or set margin/padding offset our of the container ?

Reply:
Why we need here fixed width ?

Reply:
Yeah, you are right Sir. we can rely on margins. I have now used margins only.